### ISSUE TITLE
Order.getStatus() returns null

### ISSUE THREAD

**Issue by FreeGoldRush** [studerw/td-ameritrade-client#20]
Immediately after calling client.placeOrder() a call to order.getStatus returns null.  This is unexpected.  Shouldn't it return a Status instance?  The call to placeOrder() places an order and seems to work as expected.

**Comment by studerw** [comment]
Yeah. That’s how it works in the integration tests. Can you try and sleep for a few seconds and then call the getStatus? Maybe TDA has a synchronization delay between their internal servers.

**Comment by studerw** [comment]
@FreeGoldRush Did the sleeping for a few seconds solve your problem?

**Comment by FreeGoldRush** [comment]
Sorry, have not tried it yet.  Sleeping isn't an option.  I'll probably just create a Thread that checks order status every 250 ms or something until it gets a valid value.
